nat/interface.go

```diff
@@ -19,9 +19,10 @@ package nat
 
 // NATService describes fake nat service for darwin
 type NATService interface {
-	Add(rule RuleForwarding)
-	Start() error
-	Stop()
+	Enable() error
+	Add(rule RuleForwarding) error
```

Comment:
We return an error, but swallow it in the calling code. We should print out a warning, since it might be not a real error.

Reply:
Fixed error handling. Thanks for catching it.